internal/expr/prepare.go

```diff
@@ -60,22 +60,27 @@ func starCount(fns []fullName) int {
 	return s
 }
 
+type typeNameToInfo map[string]typeInfo
+
+func typeNotPassedError(typeName string, allTypeNames []string) error {
```

Comment:
Can we please rename the parameters to `missingType` and `existingTypes`, to make the function more clear?

Also, maybe the function name migth be shorter?  typeMissingError?

Reply:
Changed, I think those do make it clearer.